cypress/integration/unit/settings/general/shouldTestSettingsOverviewPage.spec.js

```diff
@@ -69,27 +69,35 @@ context('Settings overview page tests', () => {
     cy.closeSettingsModal();
   });
 
-  it('Upload a users CSV', () => {
-    cy.route('/');
+  it('Upload a CSV and delete a user', () => {
+    cy.route('GET','/users/*').as('getUsers')
+    cy.route('GET', '/users?filter=**').as('filterUsers');
+    cy.visit('/');
     cy.get(toolbar.settings).click();
     cy.url().should('include','instellingen/overzicht');
     cy.get(settings.manageUsers).contains('Gebruikersbeheer').click();
     cy.url().should('include','instellingen/gebruikers');
     cy.contains('Gebruikers importeren').click();
-    cy.uploadUsersFile('files','test', 'csv');
-    cy.get(settings.settingsUserTable).contains('Wendy');
-  });
-
-  it('Should delete user', () => {
-
-    cy.route('GET', '/users/*').as('getUsers');
-
-    cy.get(settings.manageUsers).contains('Gebruikersbeheer').click();
-    cy.url().should('include','instellingen/gebruikers');
-    cy.get(settings.deleteUser).should('exist').should('be.visible').eq(1).click();
+    cy.uploadUsersFile('files','importUsers', 'csv');
+    cy.get(settings.userSearchInput).type("Wendy");
+    cy.get(settings.userSearchButton).click().wait('@filterUsers');
+    cy.get(settings.settingsUserTable).contains('Wendy').parents('tr').within(() => {
+      cy.get(settings.deleteUser).should('exist').should('be.visible').click();
+    });
     cy.get(modal.verify.save).should('exist').should('be.visible').click();
     cy.wait('@getUsers').then(() => {
-      cy.get(settings.settingsUserTable).should('not.have.value','Greta');
+      cy.get(settings.settingsUserTable).should('not.have.value','Wendy');
+    });
+    cy.get(settings.userSearchInput).clear();
+    cy.get(settings.settingsUserTable).contains('Greta').parents('tr').within(() => {
+      cy.contains('overheid');
+    });
+    cy.uploadUsersFile('files','updateUserGroup', 'csv');
+    cy.get(settings.userSearchInput).type("Greta");
+    cy.get(settings.userSearchButton).click().wait('@filterUsers');
```

Comment:
Best voor elke occurence van @filterusers

cy.route('GET', '/users?filter=**').as('filterUsers');
ACTION
cy.wait('@filterUsers');

Da's iets properder, staat ook zo in de docs van cypress

Reply:
Done